README.md

```diff
@@ -31,23 +34,59 @@ To run in watch mode, add `-w`/`--watch` to the command:
 steiger ./src --watch
 ```
 
-# Configuration
+## Configuration
 
 Steiger is zero-config! If you don't want to disable certain rules, you can safely skip this section.
 
 Steiger is configurable via `cosmiconfig`. That means that you can create a `steiger.config.ts` or `steiger.config.js` file in the root of your project to configure the rules. Import `{ defineConfig } from "steiger"` to get autocompletion.
 
-```ts
-import { defineConfig } from 'steiger'
+The config file shape is highly inspired by ESLint's config file, so if you have configured ESLint before, you'll find it easy to configure Steiger.
+
+### Example
+
+Here are some rules on how configuration is processed:
+
+- Config objects are processed from top to bottom, so if there are multiple config object that match the same file for the same rule, the last one will be applied.
```

Comment:
issue: I think it might be slightly unclear what "config objects" are, they might be confused with the entire file.

suggestion:
```suggestion
- Objects in the config array are processed in order, so later objects can override the objects before them.
```